Tooling-Landscape/Unanimous-Understanding/OSS-Tooling-Landscape-Glossary.md

```diff
@@ -3,46 +3,66 @@
 
-### Security Vulnerabilities
+### Security Vulnerability Database
 Systems or services that provide information about publicly known security vulnerabilities of software artefacts.
 
-### License Obligations
-Systems or services that provide (machine readable?) Data about obligations and prohibitions of (FOSS) licenses.
+### Public License Master Database
+Publicly available information about Open Source licenses (e.g. commonly accepted license identifiers). It could furthermore provide (potentially machine readable) data about the corresponding obligations and prohibitions of a license.
 
-### Public License Master Data
-Publicly available information about Open Source licenses (e.g. commonly accepted license identifiers). 
-
-### FOSS Meta Data
+### FOSS Metadatabase
 Systems or services that provide descriptive data of software artefacts (mainly FOSS). Data could be: the declared license for a component, the source location (e.g., Git commit) for a version, details to be included in attributions (e.g., copyright holders in a Notices file), etc.
 
 ## Company Internal
 
-### CI/CD Pipeline
-A system that enables automating the creation of a software build and the associated processes including: compiling source code into binary code, packaging binary code,and running automated tests. 
-
 ### Source Code Repository
-Systems or services that provide source code. These systems or services can provide plain source code only or – in addition – also have version control. 
+Systems or services that provide source code. This is typically a version control system. For the compliance tooling, typically the source code is needed as a folder on a file share, i.e., the source code is already checked out by, e.g., the Continuous Integration infrastructure. 
+
+### Continuous Integration Infrastructure
+Systems or services that orchestrate the build process for a software project and executes workflows triggered by different kind of events. The CI infrastructure typically runs software builds and executes further build steps like the compliance tooling
+
+### Build Technology
+A system that builds a software project and creates the binaries and executables for the software. During this process, the build technology has a technology dependent way to identify and provide dependencies needed to build and run the software. This information is needed during the compliance tooling workflow to run the compliance check on the project.
+
+### Compliance Tooling
+A set of systems or services that are executed on the source code of a project to identify third party software, especially FOSS components, to provide the metadata for the identified components and to run a compliance check according to defined criteria, such as license compatibility, use if software with security violations.
+
```

Comment:
Actually, license scanner, binary scanner, etc. appears to be more appropriate for an entity.